cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -354,5 +355,18 @@ class Login_Management {
 			}
 		});
 	}
+
+	assertLoginFormIsVisible(namespace) {
+		if (namespace === "nbc") {
+			cy.get(Login_Management.#buttonLoginViaEmailNbc).should("be.visible");
+		} else if (namespace === "dbc") {
+			cy.get(Login_Management.#openLoginButtonDBC).should("be.visible");
+			// ggf. weitere Checks für dbc
+		} else if (namespace === "brb") {
+			cy.get(Login_Management.#loginFormSelector).should("be.visible");
+		} else {
+			cy.get(Login_Management.#loginFormSelector).should("be.visible");
```

Comment:
this is redundant condition, I think second 'else if' in line #365 is not needed, instead this could be:

if (namespace === "nbc") {
        cy.get(Login_Management.#buttonLoginViaEmailNbc).should("be.visible");
    } else if (namespace === "dbc") {
        cy.get(Login_Management.#openLoginButtonDBC).should("be.visible");
    } else {
        // ggf. weitere Checks für brb
        cy.get(Login_Management.#loginFormSelector).should("be.visible");
    }